File: Lógica de Programação Java e C#/3 - Operadores/Java/Exercícios Complementares/TestaTelefone.java
class TestaTelefone{
	public static void main(String [] args){
	String telefone="2387-3791";
	System.out.println(telefone);
	String a = (telefone.substring(0,2));
	int primeiro = Integer.parseInt(a);
	System.out.println(primeiro);
	primeiro++;
	System.out.println(primeiro);
	a = (telefone.substring(2,4));
	int segundo = Integer.parseInt(a);
	System.out.println(segundo);
	segundo--;
	System.out.println(segundo);


	
}
}
	

File: Lógica de Programação Java e C#/3 - Operadores/C#/Exercícios Complementares/TestaTelefone.cs
class TestaTelefone{
	static void Main(){
		string telefone="2387-3791";
		System.Console.WriteLine(telefone);
		string a = (telefone.Substring(0,2));
		int primeiro = System.Convert.ToInt32(a);
		System.Console.WriteLine(primeiro);
		primeiro++;
		System.Console.WriteLine(primeiro);
		a = (telefone.Substring(2,2));
		int segundo = System.Convert.ToInt32(a);
		System.Console.WriteLine(segundo);
		segundo--;
		System.Console.WriteLine(segundo);	
	}
}
	